benchmarks/integer_overflow_and_underflow/README.md

```diff
@@ -1,15 +1,14 @@
-# Title 
-Integer Overflow and Underflow
+# Title
+Uint Integer Overflow via addition
```

Comment:
My idea was that this becomes a generic issue description and not the description for the test case itself. Can me move specific information about the issue in the code itself?

Reply:
Well, in my opinion this _is_ the generic issue. The specific information is that it is found at location such using operation such and such (which here could be either `+=` or `+` in Solidity.

By the way, based on bitter experience you are not going to find me _ever_ calling a location a line.  Sorry. 

A "Uint Integer Overflow via addition" on of the general issues in the category "Integer Overflow and Underflow".  In the discussion the other day we had in zoom and where we were working on the spreadsheet, I recall there was a deliberate decision to remove the higher-order category and leave the lower version of that. And there I said okay, as long as various tools keep that separate from the informational message which fills in location and/or source-code or EVM context.

Reply:
I don't think it makes sense to have separate READMEs for "Uint Integer Overflow via addition", "Uint Integer Overflow via multiplication", and so on. We need to include *benchmarks* for each variant though.

Reply:
Sure. But that is the way this was written. The title is describing what it _is_, not what it _should _eventually be. And actually, I would prefer that to always be the case. 

So sure, fix the code, then fix that title.

But this leads to another issue we'll probably be getting to soon enough when there are 10 issues in a benchmark and we address some fraction of those. It kind of sucks to say you failed everything when you have failed only some of them. 

Suhabe and (Not so) Smart Contracts are at different ends of that spectrum. Think unit tests vs integration tests. 

My take is just as in testing, you have _both_. Or here we have several sets of benchmarks. 

In that sense, Omnibus isn't really Omnibus. It is just one of a bunch of benchmark tests.

Reply:
> It kind of sucks to say you failed everything when you have failed only some of them.

Fair enough, but then we're getting into where reporting becomes more complex. One thing we could do is counting the number of true positives, false positives and false negatives and somehow calculating a composite "accuracy score". But maybe we shouldn't complicate things too much in the beginning.